In [2]:
import pandas as pd 
import matplotlib.pyplot as plt


In [14]:
df_infos_rang_joueurs = pd.read_csv('./Data/Data_utiles/info_rank')

In [15]:
df_infos_rang_joueurs.head()


Unnamed: 0  ranking_date  rank  player_id  points name_first name_last  \
0      134199      19900101     1     100656  2913.0       Ivan     Lendl   
1      134465      19900101     2     101414  2279.0      Boris    Becker   
2      134961      19900101     3     101222  2111.0     Stefan    Edberg   
3      135361      19900101     4     100763  1398.0       Brad   Gilbert   
4      135627      19900101     5     100581  1354.0       John   McEnroe   

  hand         dob  ioc  height wikidata_id  
0    R  19600307.0  USA   188.0     Q182736  
1    R  19671122.0  GER   190.0      Q76334  
2    R  19660119.0  SWE   188.0     Q189542  
3    R  19610809.0  USA   185.0     Q715134  
4    L  19590216.0  USA   180.0      Q16474

In [16]:
df_infos_rang_joueurs.shape

(162982, 12)

On voit que ce dataset est composé de $12$ colonnes et $162982$ lignes, ce qui est assez conséquent. 

Le but de notre projet étant de définir des caractéristiques qui distinguent un joueur du top 10 d'un autre du top 50, nous pouvons d'ors et déjà supprimer certaines colonnes qui ne nous seront pas utiles telles que:

- IOC vu que le pays d'origine du joueur n'a aucun impact sur ses performances
- wikidata_id
- dob
- Unnamed :0 (qui est un duplicat des indexes)

In [17]:
df_infos_rang_joueurs.drop(['Unnamed: 0', 'ioc', 'wikidata_id','dob'],axis=1,inplace=True)



In [18]:
df_infos_rang_joueurs.head()

ranking_date  rank  player_id  points name_first name_last hand  height
0      19900101     1     100656  2913.0       Ivan     Lendl    R   188.0
1      19900101     2     101414  2279.0      Boris    Becker    R   190.0
2      19900101     3     101222  2111.0     Stefan    Edberg    R   188.0
3      19900101     4     100763  1398.0       Brad   Gilbert    R   185.0
4      19900101     5     100581  1354.0       John   McEnroe    L   180.0

In [20]:
df_infos_rang_joueurs.to_csv('./Data/Data_utiles/info_rank_after_treatment')